api/v1alpha1/openapi.yaml

```diff
@@ -2529,6 +2631,22 @@ components:
               type: array
               items:
                 type: string
+        resources:
+          type: object
+          properties:
+            cpu:
+              $ref: '#/components/schemas/ResourceCPUConfigSpec'
```

Comment:
The resources and alertRules are essentially the same, right? Could we have one generic ResourceAlertRule schema that applies to all of them (plus user-defined ones)?